Modules/_cursesmodule.c

```diff
@@ -1736,6 +1736,54 @@ _curses_window_inch_impl(PyCursesWindowObject *self, int group_right_1,
     return rtn;
 }
 
+#ifdef HAVE_NCURSESW
+/*[clinic input]
+_curses.window.in_wch
+
+    [
+    y: int
+        Starting Y-coordinate.
+    x: int
+        Starting X-coordinate.
+    ]
```

Comment:
```suggestion
    y: int
        Y-coordinate.
    x: int
        X-coordinate.
    ]
```

Let's use the same wording as for `inch`.